ui/app/components/mount-backend/type-form.hbs

```diff
@@ -11,7 +11,7 @@
     {{#each (filter-by "category" category this.mountTypes) as |type|}}
       <SelectableCard
         id={{type.type}}
-        class="has-top-padding-m has-text-centered small-card"
+        class="has-top-padding-l has-text-centered small-card"
```

Comment:
had to increase when font sizes increased
<img width="777" alt="Screenshot 2023-11-03 at 9 28 50 AM" src="https://github.com/hashicorp/vault/assets/68122737/e82b165a-0299-40af-9c43-e49c3643c671">